src/components/step-sequence/step-sequence.style.js

```diff
@@ -0,0 +1,29 @@
+
+import styled, { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+import StepSequenceClassicStyle from './step-sequence-classic.style';
+import StepSequenceItemVerticalStyle from './step-sequence-item/step-sequence-item-vertical.style';
+import StepSequenceItemVerticalClassicStyle from './step-sequence-item/step-sequence-item-vertical-classic.style';
+
+const StepSequenceStyle = styled.ol`
+  display: flex;
+  margin: 0;
+  padding: 18px;
+  font-weight: bold;
+  
+  ${({ orientation }) => orientation === 'vertical' && css`
+    flex-direction: column;
+    
+    /* Avoids passing orientation prop down to child items of StepSequence */
+    ${StepSequenceItemVerticalStyle}
+  `};
+  
+  ${({ theme, orientation }) => theme.name === THEMES.classic && css`
+    ${StepSequenceClassicStyle}
+    
+    /* Avoids passing orientation prop down to child items of StepSequence */
```

Comment:
Does this comment render inside of the css which is actually loaded in the browser? You might want to put it outside of the template string, and maybe be a little more explicit about what you're achieving here.